backend/mobilityAI/Controllers/MobilityAIController.cs

```diff
@@ -27,9 +29,13 @@ public class MobilityAIController : ControllerBase
     {
         const string ML_SERVER_URL = "http://127.0.0.1:6000/";
         const string SERVER_URL = "http://127.0.0.1:5000/";
+        const string SERVER_SECURE_URL = "https://127.0.0.1:5001/";
         private readonly MobilityAIContext _context;
         private static readonly HttpClient client = new HttpClient();
         private static ConcurrentDictionary<string, int> mlCallbackIds = new ConcurrentDictionary<string, int>();
+
+        private static readonly DateTime epoch = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
```

Comment:
Not using this either

Reply:
Should be used later on, see my new commit.